lib/rules/indent.js

```diff
@@ -807,6 +889,71 @@ module.exports = {
             });
         }
 
+        /**
+        * Ignore first token of each line within an unknown node if its
+        * offset does not depend on another token's offset. If the first token
+        * of a line's offset depends on the offset of another token, make the
+        * last dependent offset depend on the first token of its line.
```

Comment:
I realize the behavior is a bit hard to explain, but I'm confused by the last sentence of this comment. What does it mean to "make the last dependent offset depend on the first token of its line"? Is this supposed to say "make the last dependent token's offset depend on the first token of it's line"?